authorization/role/scope/repository/role_scope.go

```diff
@@ -0,0 +1,129 @@
+package repository
```

Comment:
Also, I thought we decided to flatten our structure during last week's call, so shouldn't this just be in the authorization/role/repository directory?

Reply:
I remember that we agreed on this too.

Reply:
yeah, had missed doing so fccf380bc0733dae57d5a6f15145cd638cfa77df